multinet/validation/csv.py

```diff
@@ -1,29 +1,25 @@
 """Utilities for validating tabular data for upload to Multinet."""
 
-from dataclasses import dataclass
 import re
 from typing import Set, MutableMapping, Sequence, List
 
 from multinet.errors import ValidationFailed
 from multinet.validation import ValidationFailure, DuplicateKey, UnsupportedTable
 
 
-@dataclass
 class InvalidRow(ValidationFailure):
     """Invalid syntax in a CSV file."""
 
     row: int
-    fields: List[str]
+    columns: List[str]
```

Comment:
I renamed this because apparently `fields` is a reserved keyword in pydantic, and I figured `columns` was just as representative of the underlying data, if not more.